network/definition/data/components/vcfront/vcfront-message.yaml

```diff
@@ -74,3 +74,17 @@ messages:
     signals:
       5vCriticalHsdState:
       5vExtHsdState:
+
+  wheelSpeed:
+    description: Front wheel speeds
+    id: 0x461
+    cycleTimeMs: 50
+    signals:
+      velocityL:
+        template: tireLinearVelocity
+      velocityR:
+        template: tireLinearVelocity
+      rpmL:
+        template: tireRPM
+      rpmR:
+        template: tireRPM
```

Comment:
Do you actually need near velocity for something?

IMO these signal names aren't great. `VCFRONT_velocityL` and such... Maybe `VCFRONT_wheelVelocityL` and `VCFRONT_wheelSpeedL` or `VCFRONT_wheelRpmL`?